frontend/coprs_frontend/coprs/forms.py

```diff
@@ -735,8 +735,8 @@ class F(CoprForm):
             def selected_chroots(self):
                 return self.chroots.data
 
-            def validate(self):
-                if not super(F, self).validate():
+            def validate(self, extra_validators=None):
+                if not super(F, self).validate(extra_validators):
```

Comment:
Considering the old WTForms (our current deployment where the argument doesn't exist, yet) -> is it safe to pass None here?  I'm afraid it is some other argument actually.

Reply:
I know what you mean, I was a bit concerned as well. But I just checked

The `validate` method is not defined in `python3-flask-wtf`. The class hierarchy is funny.

    F < CoprForm < BaseForm < FlaskForm < Form < BaseForm
    These are ours          | flask-wtf |     wtforms

The `validate` method is defined in `BaseForm` in `python3-wtforms`, and it already takes `extra_validators`.

```
[root@copr-fe ~][PROD]# cat /usr/lib/python3.11/site-packages/wtforms/form.py |grep "def validate"
    def validate(self, extra_validators=None):
    def validate(self, extra_validators=None):
```

Reply:
so can we do "extra_validators=extra_validators" instead, just to be sure?

Reply:
Good idea, done.